lib/Mojo/Server/Morbo/Backend/Poll.pm

```diff
@@ -0,0 +1,53 @@
+package Mojo::Server::Morbo::Backend::Poll;
+
+use Mojo::Base 'Mojo::Server::Morbo::Backend';
```

Comment:
In all other classes the `Mojo::Base` import is at the top, separate from all other modules used.